src/main/java/frc/robot/hardware/motor/sparkmax/SparkMaxMotor.java

```diff
@@ -30,11 +32,28 @@ public boolean isConnected() {
 		return connectedInput.connected;
 	}
 
+	private boolean isValid(InputSignal<?> signal) {
+		return signal instanceof SuppliedDoubleSignal || signal instanceof SuppliedAngleSignal;
+	}
+
+	private void reportInvalidSignal(InputSignal<?> invalidSignal) {
+		new Alert(
+			Alert.AlertType.WARNING,
+			logPath + "signal named " + invalidSignal.getName() + " got invalid type " + invalidSignal.getClass().getSimpleName()
+		).report();
+	}
+
 	@Override
 	public void updateSignals(InputSignal<?>... signals) {
```

Comment:
```suggestion
	public void updateInputs(InputSignal<?>... signals) {
```

Reply:
Why?

Reply:
Because this function is used not just to refresh the signals but also other inputs such as connected input

Reply:
but i update signals, not inputs like there is in digital input auto log inputs that u cant enter here

Reply:
The need is more clear in talonfx

Reply:
First of all "signals are inputs".
Secondly connected input is an AutoLogged input